### ISSUE TITLE
Center Loss

### ISSUE THREAD

**Issue by youwei2567** [wvinzh/WS_DAN_PyTorch#5]
你这里center loss定义了，但是在训练的时候没加上去：
`    # define loss
    criterion = torch.nn.CrossEntropyLoss()
    if use_gpu:
        criterion = criterion.cuda()`
这可能就是你离[官方代码(569-572行)](https://github.com/tau-yihouxiang/WS_DAN/blob/master/train_sample.py)差一点的原因啊
